frontend/apps/picasso/defi/polkadot/pallets/Assets/picasso.ts

```diff
@@ -13,13 +13,16 @@ export async function picassoAssetsList(
 ): Promise<PicassoRpcAsset[]> {
   try {
     const assetsList = await api.rpc.assets.listAssets();
-    return assetsList.map(asset => {
+    return assetsList.map((asset) => {
       return {
         name: asset.name.toUtf8(),
         id: new BigNumber(asset.id.toString()),
-        decimals: asset.decimals.toNumber(),
+        decimals:
+          asset.name.toUtf8().toUpperCase() !== "USDT"
+            ? asset.decimals.toNumber()
+            : 6, // TODO: Temporary assign 6 decimals to USDT until RPC is ready
```

Comment:
You will manually update it to `6` then no?

Reply:
Yes, this is just for now until your PR is merged.